pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -158,17 +158,25 @@ func (c *controller) Get(typ, name, namespace string) (*model.Config, bool) {
 		return nil, false
 	}
 
-	rules := convertIngress(*ingress, c.domainSuffix)
-	for _, rule := range rules {
-		if rule.Name == name {
-			return &rule, true
+	switch typ {
+	case model.VirtualService.Type:
+		return nil, false
+	case model.Gateway.Type:
+		return nil, false
```

Comment:
nit: just remove these cases and replace with a single `if typ == model.IngressRule.Type {`